Code/RDBoost/Wrap.h

```diff
@@ -18,6 +18,10 @@
 //
 #include <boost/python.hpp>
 #include <boost/python/stl_iterator.hpp>
+#include <memory>
+// ha ha... we don't actually want to use this, but we need to keep support old
+// C++ versions:
+#define rdk_unique_ptr std::auto_ptr
```

Comment:
Yeah, I'm aware that auto_ptr's have some unusual behavior and I have no intention of starting to make really broad use of them. In this particular case it is (I believe) exactly what we're looking for: ownership is transferred to the caller.